users/views.py

```diff
@@ -255,6 +263,29 @@ def form_invalid(self, form: PasswordChangeView.form_class) -> str:
         return super().form_invalid(form)
 
 
+class CustomPasswordChangeOnLoginView(PasswordChangeView, TemplateResponseMixin, FormMixin):
```

Comment:
I won't go deep in this view because you're working on merging it now

Reply:
Ok.